pipeline-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -7,13 +7,14 @@ def branches = [:]
 //and jenkins will unite them into a single run of the job
 
 for (int i = 0; i < 4; i++) {
+  def _i = i //if we tried to use i below, it would equal 4 in each job execution.
```

Comment:
Maybe ping @jglick ?
Not totally sure myself how much of this is due to Groovy per-se or to Pipeline/CPS.